foodwebsite/style.scss

```diff
@@ -0,0 +1,361 @@
+
+$font-stack: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+$pimary-color:#FFF;
+
+.hb-header{
+    display: flex;
+    flex-flow: column wrap;
+    .header-wrapper-top{
+        height:55vh;
+        background-image: url('img/ChatGPT\ Image\ Jul\ 16\,\ 2025\,\ 09_54_09\ AM.png');
```

Comment:
it's easier to rename the file and remove the spaces, it ensures cross-os compatibility